spyder/plugins/projects/plugin.py

```diff
@@ -47,11 +48,13 @@ def __init__(self, parent=None):
         SpyderPluginWidget.__init__(self, parent)
 
         self.explorer = ProjectExplorerWidget(
-                            self,
-                            name_filters=self.get_option('name_filters'),
-                            show_all=self.get_option('show_all'),
-                            show_hscrollbar=self.get_option('show_hscrollbar'),
-                            options_button=self.options_button)
+            self,
+            name_filters=self.get_option('name_filters'),
+            show_all=self.get_option('show_all'),
+            show_hscrollbar=self.get_option('show_hscrollbar'),
+            options_button=self.options_button,
+            single_click_to_open=CONF.get('explorer', 'single_click_to_open'),

```

Comment:
I think we should a separate option for this, instead of using the same of the File explorer. What do you think?

Reply:
Well, I think the user that reported the issue was expecting the behavior on the file explorer to be the same on the project explorer, so it would make sense to use a single configuration value.

That is what I would expect as well. I think it would be weird to have single click on one vs double click on the other.

Reply:
Ok, fair enough. But in that case you need to make the File Explorer emit `sig_option_changed` when the `single_click_to_open` changes and apply the corresponding changes in the project explorer.

Else switching from single to double click wouldn't take effect immediately in the project explorer, but only until the next restart.